pycryptobot.py

```diff
@@ -1032,7 +1032,7 @@ def executeJob(sc, app=PyCryptoBot(), state=AppState(), trading_data=pd.DataFram
 def main():
     try:
         # initialise logging
-        logging.basicConfig(filename=app.getLogFile(), format='%(asctime)s - %(levelname)s: %(message)s', datefmt='%m/%d/%Y %I:%M:%S %p', filemode='a', level=logging.DEBUG)
+        logging.basicConfig(filename=app.getLogFile(), format='%(asctime)s - %(levelname)s: %(message)s', datefmt='%m/%d/%Y %I:%M:%S %p', filemode='a', level=logging.DEBUG, force=True)
```

Comment:
From what I understood, there were logging handlers in place before reaching this point, so the file was not created because of it.

The [documentation ](https://docs.python.org/3/library/logging.html#logging.basicConfig)says that

> If this keyword argument is specified as true, any existing handlers attached to the root logger are removed and closed, before carrying out the configuration as specified by the other arguments.

Reply:
Have you tested this? You sure this works?

Reply:
Yes. I use the bot on my raspberry pi so I don't usually see the print statements, the only way for me to see what's going on with the bot is by looking at the logs, so I have the bots working now with this change locally.

Reply:
Sorry I missed that during my refactoring.
I did test and it does work.

I was thinking that we should not have to do so, but not sure how that handle the issue.

Reply:
The line is a bit long (185char)
maybe we should break it
```python
        logging.basicConfig(
            filename=app.getLogFile(),
            format='%(asctime)s - %(levelname)s: %(message)s',
            datefmt='%m/%d/%Y %I:%M:%S %p',
            filemode='a',
            level=logging.DEBUG,
            force=True
        )
```

Reply:
Done. Thanks